code/style.css

```diff
@@ -1,4 +1,309 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/** HEADER **/
+
+header {
+  padding: 10px 20px;
+  background: #FFFFFF;
+  color: #121212;
+  font-family: Helvetica;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
+
+@media (min-width: 768px) and (max-width: 1024px) {
+  header {
+    justify-content: space-evenly;
+  }
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  /*to spread the lines, since they are stacked*/
+  justify-content: space-between;
+  height: 12px;
+  width: 18px;
+  margin: 0px, 0px, 0px, -80px;
+}
+
+.hamburger div {
+  width: 18px;
+  height: 2px;
+  background-color: #121212;
+}
+
+@media (min-width: 768px) {
+  .hamburger {
+    display: none;
+  }
+}
+
+header img {
+  width: 130px;
+  height: 30.42px;
```

Comment:
Most of browsers will not detect that value and just round it to `30px`, since `1px` is the smallest unit. We should write them without values after dot/comma